integration-tests/go.mod

```diff
@@ -8,12 +8,12 @@ require (
 	github.com/google/uuid v1.4.0
 	github.com/lib/pq v1.10.9
 	github.com/rs/zerolog v1.30.0
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20231218150613-43bf581ae327
-	github.com/smartcontractkit/chainlink-cosmos v0.4.1-0.20231215215216-51cb121f7f33
+	github.com/smartcontractkit/chainlink-common v0.1.7-0.20231222013743-f67285bba7bd
+	github.com/smartcontractkit/chainlink-cosmos v0.4.1-0.20231222201016-da3f0a763f71
 	github.com/smartcontractkit/chainlink-cosmos/ops v0.0.0-20231206164210-03f8b219402e
 	github.com/smartcontractkit/chainlink-testing-framework v1.22.0
 	github.com/smartcontractkit/chainlink/integration-tests v0.0.0-20231215220731-a4091a3e4d3c
-	github.com/smartcontractkit/chainlink/v2 v2.8.0-beta0.0.20231215220731-a4091a3e4d3c
+	github.com/smartcontractkit/chainlink/v2 v2.8.0-beta0.0.20231222204838-fa1325e3a88c
```

Comment:
`chainlink/integration-tests` should be bumped to `fa1325e3a88c` also

Reply:
bumped

Reply:
thansk.